.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+#      - 'main'
+#      - 'master'
+#      - 'refs/heads/v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
```suggestion
```

Reply:
Almost, see the version I just pushed. it will only run on the default branch and new releases. So it doesn't run twice during PR's from the same repository.